atc/metric/emitter/prometheus_test.go

```diff
@@ -188,7 +188,9 @@ var _ = Describe("PrometheusEmitter", func() {
 	})
 
 	JustBeforeEach(func() {
-		prometheusEmitter, err = prometheusConfig.NewEmitter(nil)
+		prometheusEmitter, err = prometheusConfig.NewEmitter(map[string]string{
+			"invalid-label": "foo", // Ensure invalid labels are sanitized.
```

Comment:
Could you please add some more test strings here e.g. _blah__

Just to cover the regex `rePrometheusLabelClean`.